### PR TITLE
chore: move `commonDb` into  `diagrams/common/commonDb`

### PR DESCRIPTION
**Pull request by Yokozuna59**
## :bookmark_tabs: Summary

Move and change default export to named one.

### :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

File: packages/mermaid/src/diagrams/common/commonDb.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+export const db: CommonDb = {
+  getAccTitle,
+  setAccTitle,
+  getDiagramTitle,
+  setDiagramTitle,
+  getAccDescription,
+  setAccDescription,
+  clear,
+};
```
Comment (sidharthv96) [inline packages/mermaid/src/diagrams/common/commonDb.ts:47]: I don't see this being used anywhere. Maybe we can remove the db export?
Also, if the plan is to keep the export, rename db to commonDB.

File: packages/mermaid/src/diagrams/common/commonTypes.ts [hunk 2]
```diff
@@ -56,3 +58,13 @@ export type D3ImageElement = d3.Selection<SVGImageElement, unknown, Element | nu
 export type D3TextElement = d3.Selection<SVGTextElement, unknown, Element | null, unknown>;
 
 export type D3TSpanElement = d3.Selection<SVGTSpanElement, unknown, Element | null, unknown>;
+
+export interface CommonDb extends DiagramDB {
```
Comment (sidharthv96) [inline packages/mermaid/src/diagrams/common/commonTypes.ts:62]:
```suggestion
export interface CommonDB extends DiagramDB {
```

**Review by sidharthv96 — CHANGES_REQUESTED** [review]

File: packages/mermaid/src/diagrams/common/commonDb.ts (hunk 1 above)
Comment (Yokozuna59) [reply packages/mermaid/src/diagrams/common/commonDb.ts:47]: I saw that too, all diagrams uses named import or `import * as commonDb`. I think we could just remove it.

File: packages/mermaid/src/diagrams/common/commonTypes.ts (hunk 2 above)
Comment (Yokozuna59) [reply packages/mermaid/src/diagrams/common/commonTypes.ts:62]: Since we will remove the `db`, should we remove this also?

**Pull request merged**